client/src/pages/club/match/CreateMatch.tsx

```diff
@@ -50,34 +52,92 @@ export interface TeamList {
   members: string[];
 }
 
-function CreateMatch() {
-  const [isOpenMapSetting, setIsOpenMapSetting] = useState<boolean>(false);
-  const [isOpenMapView, setIsOpenMapView] = useState<boolean>(false);
-  const [placeValue, setPlaceValue] = useState<PlaceType>();
+export interface Record {
+  id: number;
+  firstTeam: string;
+  secondTeam: string;
+  firstTeamScore: number;
+  secondTeamScore: number;
+}
 
-  const [teamList, setTeamList] = useState<TeamList[]>([
-    {
-      id: 0,
-      members: []
-    }
-  ]);
+export interface Datas {
+  schedule: {
+      date: string | undefined;
+      time: string | undefined;
+      placeName: string | undefined;
+      placeCoordinate: {
+          longitude: number | undefined;
+          latitude: number | undefined;
+      };
+  };
+  canidates: string[];
+  teamList: TeamList[];
+  records: Record[];
+}
 
-  //match type을 BE table 구조를 보고 정확하게 정하지 못한상태여서 any타입으로 임시지정.
-  const [matchResult, setMatchResult] = useState<any>([]);
+function CreateMatch() {
+  const {
+    register,
+    unregister,
+    // formState: { errors },
+    getValues,
+} = useForm({ mode: "onChange" });
 
-  const candidates: string[] = ['박대운', '우제훈', '김은택', '김아애', '문채리', '전규언'];
+  const [datas, setDatas] = useState<Datas>();
 
-  //팀구성에 필요한 후보들
+  const [date, setDate] = useState<string>();
+  const [time, setTime] = useState<string>();
+  const [placeValue, setPlaceValue] = useState<PlaceType>();
+  //참가를 누른 멤버들
+  const candidates: string[] = ['박대운', '우제훈', '김은택', '김아애', '문채리', '전규언전'];
+  
+  //팀구성에 필요한 후보들(팀에 들어가거나 빠질 때 실시간 반영되는 리스트)
   const [candidateList, setCandidateList] = useState(candidates);
 
+  const [teamList, setTeamList] = useState<TeamList[]>([{ id: 0, members: [] }]);
+  const [records, setRecords] = useState<Record[]>([]);
+  // const records:Record[] = [];
+
+  
+
+  const [isOpenMapSetting, setIsOpenMapSetting] = useState<boolean>(false);
+  const [isOpenMapView, setIsOpenMapView] = useState<boolean>(false);
```

Comment:
useState 원시형 초기값 설정 해주면 <boolean> 없이 가는 걸로 ts 코드 컨벤션 정한 부분 반영해 주시면 좋을 것 같습니다. 

```suggestion
  const [isOpenMapSetting, setIsOpenMapSetting] = useState(false);
  const [isOpenMapView, setIsOpenMapView] = useState(false);
```